DESCRIPTION

```diff
@@ -29,5 +29,5 @@ Config/testthat/edition: 3
 License: GPL-2
 Encoding: UTF-8
 LazyData: true
-RoxygenNote: 7.3.1
+RoxygenNote: 7.3.2
```

Comment:
Any particular reason why this incremented?

Reply:
It was done automatically by `devtools::document()`, possibly I updated Roxygen since the last time, not sure though.